test/OpenTelemetry.Contrib.Instrumentation.MassTransit.Tests/MassTransitInstrumentationTests.cs

```diff
@@ -55,33 +53,173 @@ public async Task MassTransitInstrumentationConsumerAndHandlerTest()
                 {
                     await harness.Stop();
                 }
-            }
 
-            Assert.Equal(10, activityProcessor.Invocations.Count);
+                var expectedMessageContext = harness.Sent.Select<TestMessage>().FirstOrDefault()?.Context;
+                var actualActivity = this.GetActivitiesFromInvocationsByOperationName(activityProcessor.Invocations, OperationName.Transport.Send).LastOrDefault();
 
-            var sends = this.GetActivitiesFromInvocationsByOperationName(activityProcessor.Invocations, OperationName.Transport.Send);
-            var receives = this.GetActivitiesFromInvocationsByOperationName(activityProcessor.Invocations, OperationName.Transport.Receive);
-            var consumes = this.GetActivitiesFromInvocationsByOperationName(activityProcessor.Invocations, OperationName.Consumer.Consume);
-            var handles = this.GetActivitiesFromInvocationsByOperationName(activityProcessor.Invocations, OperationName.Consumer.Handle);
+                Assert.NotNull(actualActivity);
+                Assert.NotNull(expectedMessageContext);
 
-            foreach (var activity in sends)
-            {
-                Assert.Equal("SEND /input_queue", activity.DisplayName);
+                Assert.Equal("/input_queue send", actualActivity.DisplayName);
+                Assert.Equal(ActivityKind.Producer, actualActivity.Kind);
+                Assert.Equal("loopback", actualActivity.GetTagValue(SemanticConventions.AttributeMessagingSystem)?.ToString());
```

Comment:
This tag could be `rabbitmq`, `activemq`, etc for all real providers.
But for the in-memory provider, we have `loopback`. Is it OK?